src/Geometry.cc

```diff
@@ -327,6 +351,9 @@ std::optional<gz::math::Inertiald> Geometry::CalculateInertial(
     case GeometryType::CAPSULE:
       geomInertial = this->dataPtr->capsule->CalculateInertial(_density);
       break;
+    case GeometryType::CONE:
+      geomInertial = this->dataPtr->capsule->CalculateInertial(_density);
```

Comment:
```suggestion
      geomInertial = this->dataPtr->cone->CalculateInertial(_density);
```

Reply:
Making this fix to harmonic PR as well: https://github.com/gazebosim/sdformat/pull/1415/commits/7e96871ab7cb6a660ee0429dbb68a1fff1a693cb